app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1835,6 +1837,11 @@ class BrowserTabFragment :
             hideHomeCta()
             hideHomeTopCta()
             configuration.showCta(daxCtaContainer)
+            newTabLayout.setOnClickListener { daxCtaContainer.dialogTextCta.finishAnimation() }
+        }
+
+        private fun resetNewTabLayoutClickListener() {
```

Comment:
what about `removeNewTabLayoutClickListener`? reset sounds more like you are going to set a new one instead of just removing the listener.

Reply:
Makes sense, done!